zadaci/sa-kolokvijuma/2020/PSI/T5/G7/zadatak.rst

```diff
@@ -0,0 +1,42 @@
+Paketi su opisani na sledeći način:
+
+* sadržaj (jedna reč, maksimalno 30 karaktera)
+* masa (realna vrednost, u kilogramima)
+* lomljivo (jedna reč, "DA", ili "NE", maksimalno 2 karaktera)
+  
+Paketi se šalju putem kurirske službe, koja naplaćuje usluge slanja paketa u odnosu na njegovu masu.
+Učitati pakete u jednostruko spregnutu listu (dodati novi element na kraj liste) i uraditi sledeće:
+
+a) Izračunati cenu dostave paketa na osnovu sledećeg cenovnika:
+
++-----------+------+
```

Comment:
Please nest/indent this table (and the next paragraph), so they belong to the `a)` list item above.

Reply:
Not yet fixed, see the render on https://github.com/petarmaric/pjisp-zbirka-zadataka/blob/e7680e6fb6655b6972af37b4fd3a6444a008616f/zadaci/sa-kolokvijuma/2020/PSI/T5/G7/zadatak.rsta

Reply:
I have to include these snippets from zbirka.pdf: 
![image](https://user-images.githubusercontent.com/12072588/116155917-14b71400-a6eb-11eb-8a7a-0e5d29705f4e.png)

![image](https://user-images.githubusercontent.com/12072588/116156504-dec65f80-a6eb-11eb-8846-f6d2e3d44e8b.png)

and zbirka.html:

![image](https://user-images.githubusercontent.com/12072588/116156605-fef61e80-a6eb-11eb-8d96-a896db2670cf.png)

![image](https://user-images.githubusercontent.com/12072588/116156259-842d0380-a6eb-11eb-9623-0dd25dd8efd2.png)

Seems like RestructuredText elements are rendered differently on different outputs... Don't know if it's a bug or a feature, but I don't like it. :( I had similar problem with nested indent before, I just need to remember how did I fix it.

Reply:
I remembered it now: text must be aligned after a bullet or a numbered list, like this:

![image](https://user-images.githubusercontent.com/12072588/116158687-5ea1f900-a6ef-11eb-881d-f0006ed624da.png)